storage/src/factory.rs

```diff
@@ -74,16 +77,26 @@ pub struct BlobFactory {
 impl BlobFactory {
     /// Create a new instance of blob factory object.
     pub fn new() -> Self {
-        BlobFactory {
+        let blobfactory = BlobFactory {
             mgrs: Mutex::new(HashMap::new()),
-        }
+        };
+
+        ASYNC_RUNTIME.spawn(async {
```

Comment:
Sorry. I didn't point out this. But I think it's better to move this timer into `FsCacheMgr::new` since FUSE daemon does not need the timer.

Reply:
Yes , make sense , will fix it.

Reply:
fixed & updated